test/mockEvent.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var test = require('tape');
+
+var mockEventCreator = require('../mockEvent.js');
+
+
+test('mockEventCreator - createDynamoDBEvent', function(t) {
+  t.test('no options: returns an object with 1 dynamo "INSERT" event, with the default eventSourceARN and an awsRegion of "eu-west-1"', function(st) {
```

Comment:
@nelsonic neither am i. i didnt get it at first. In the end I went with keeping consistency